recipe/meta.yaml

```
@@ -36,6 +36,7 @@ outputs:
     build:
       ignore_run_exports:
         - cuda-version
+        - cudatoolkit
```

How does `cudatoolkit` come up here? Could you please share some more context?

---

What if we wrote this like so?

```suggestion
      ignore_run_exports_from:
        - {{ compiler("cuda") }}
```

---

It's an export from the CTK11 toolchain. We don't want to ignore all run_exports from the compile because it could also export arm-variant.

---

I am removing the dependency on cudatoolkit from the CTK11 variant of this package because conda does not detect any links from here to cudatoolkit and I don't want that package in the dependency tree unnecessarily.